index.html

```diff
@@ -21,6 +21,12 @@
 	          <div><a href="#about">About</a></div>
 	          <div><a href="#contact">Contact</a></div>
             </div>
+            <div class="mobile-item">
+          <div class="item close"><a href="#" class="closeMenu"><i class="fa fa-times" aria-hidden="true"></i></a></div>
+          <div class="mobile-nav"><a href="#portfolio" class="closeMenu">Portfolio</a></div>
+          <div class="mobile-nav"><a href="#about" class="closeMenu">About</a></div>
+          <div class="mobile-nav"><a href="#contact" class="closeMenu">Contact</a></div>
+        </div>
```

Comment:
- [ ] Kindly consider making menu items aligned left and make the font weight bold for items like figma design.
- [ ] Also consider adding blur effect for the background like the design.

| Your design  | Figma design|
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/53402600/188495774-36a70aad-317c-4182-badd-8ae810119387.png) | ![image](https://user-images.githubusercontent.com/53402600/188495746-1318355f-461c-4ce3-bdfe-16bd81aa781e.png) |